system-reinstall-bootc/src/podman.rs

```diff
@@ -0,0 +1,60 @@
+use std::process::Command;
+
+use super::ROOT_KEY_MOUNT_POINT;
+use crate::users::UserKeys;
+
+pub(crate) fn command(image: &str, root_key: &Option<UserKeys>) -> Command {
+    let mut podman_command_and_args = [
+        // We use podman to run the bootc container. This might change in the future to remove the
+        // podman dependency.
+        "podman",
+        "run",
+        // The container needs to be privileged, as it heavily modifies the host
+        "--privileged",
+        // The container needs to access the host's PID namespace to mount host directories
+        "--pid=host",
+        // Since https://github.com/containers/bootc/pull/919 this mount should not be needed, but
+        // some reason with e.g. quay.io/fedora/fedora-bootc:41 it is still needed.
```

Comment:
We should try to root cause some of this. Let's pile onto https://github.com/containers/bootc/issues/1053